File: program/program/reverse-an-array/ReverseAnArray.fs
// function to reverse an array
let reverseArray (arr: int array) =
    let mutable reversedArr = Array.zeroCreate arr.Length
    for i = 0 to arr.Length - 1 do
        reversedArr.[arr.Length - 1 - i] <- arr.[i]
    reversedArr

let inputArray = [|1; 2; 3; 4; 5|]
let reversedArray = reverseArray inputArray
printfn "%A" reversedArray

File: program/program/reverse-an-array/reverse_an_array.go
package main

import "fmt"

func main() {
	var arr = []int{5, 2, 3, 4, 5, 7, 8, 9}
	for i := len(arr) - 1; i >= 0; i-- {
		fmt.Print(arr[i], " ")
	}
}
